account_move_lock_date/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# © <2018> PRISEHUB CIA LTDA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Locking Dates for Account entries',
```

Comment:
`'Locking Dates for Account entries'` => `'Locking Dates for Journal Entry Posting'`